cmd/goreleaser/internal/configure.go

```diff
@@ -141,6 +143,22 @@ var (
 			newContainerImageManifests(d.name, "linux", k8sArchs, containerImageOptions{}),
 		)
 	}).WithDefaultArchives().WithDefaultChecksum().WithDefaultSigns().WithDefaultDockerSigns().WithDefaultSBOMs().Build()
+
+	// ebpf-profiler distro
+	ebpfProfilerDist = newDistributionBuilder(ebpfProfilerDistro).WithConfigFunc(func(d *distribution) {
+		d.buildConfigs = []buildConfig{
+			&fullBuildConfig{targetOS: "linux", targetArch: ebpfProfilerArchs},
+		}
+		d.containerImages = slices.Concat(
+			newContainerImages(d.name, "linux", ebpfProfilerArchs, containerImageOptions{armVersion: "7"}),
```

Comment:
[ARM7 usually](https://en.wikipedia.org/wiki/ARM_architecture_family#Cores) referees to 32 bit architectures. While ARM8 and the following support 64 bit widths.
If `"7"` referees here to the regular ARM architecture, then it should be bumped to `"8"`, as eBPF profiler is limited to 64 bit architectures.

Reply:
We don't actually need this option at all, as it would only be used for the `arm` arch. We only use `arm64`.
I have removed it.